backend/fms_core/viewsets/_constants.py

```diff
@@ -249,6 +249,13 @@
     "validation_status_timestamp": DATE_FILTERS,
 }
 
+_readset_filterset_fields: FiltersetFields = {
+    "id" : PK_FILTERS,
+    "name": CATEGORICAL_FILTERS_LOOSE,
+    "sample_name": CATEGORICAL_FILTERS_LOOSE,
```

Comment:
you should include derived_sample as FK_FILTERS since it is part of the serialized data.